README.md

```diff
@@ -0,0 +1,46 @@
+> NOTE: We have recently made significant changes to our repository structure. In order to 
+streamline our development process and foster better contributions, we have merged three separate 
+repositories Cumulus, Substrate and Polkadot into this repository. Read more about the changes [
+here](https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f).
+
+# Polkadot SDK
+
+![](https://cms.polkadot.network/content/images/2021/06/1-xPcVR_fkITd0ssKBvJ3GMw.png)
+
+[![StackExchange](https://img.shields.io/badge/StackExchange-Community%20&%20Support-222222?logo=stackexchange)](https://substrate.stackexchange.com/)
+
+The Polkadot SDK repository provides all the resources needed to start building on the Polkadot 
+network, a multi-chain blockchain platform that enables different blockchains to interoperate and 
+share information in a secure and scalable way. The Polkadot SDK comprises three main pieces of 
+software:
+
+## [Polkadot](./polkadot/)
+[![StackExchange](https://img.shields.io/badge/Polkadot_Forum-e6007a?logo=polkadot)](https://substrate.stackexchange.com/) [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](#LICENSE)
+
+Implementation of a https://polkadot.network node in Rust based on the Substrate framework. This 
+directory contains runtimes for the Polkadot, Kusama, and Westend networks. 
+
+## [Substrate](./substrate/)
+ [![SubstrateRustDocs](https://img.shields.io/badge/Rust_Docs-Substrate-24CC85?logo=rust)](https://paritytech.github.io/substrate/master/substrate/index.html) [![Substrate-license](https://img.shields.io/badge/License-GPL3%2FApache2.0-blue)](#LICENSE)
+
+Substrate is the primary blockchain SDK used by developers to create the parachains that make up 
+the Polkadot network. Additionally, it allows for the development of self-sovereign blockchains that operate completely independently of Polkadot.
+
+## [Cumulus](./cumulus/)
+[![CumulusRustDocs](https://img.shields.io/badge/Rust_Docs-Cumulus-222222?logo=rust)](https://paritytech.github.io/cumulus/cumulus_client_collator/index.html) [![Cumulus-license](https://img.shields.io/badge/License-GPL3-blue)](#LICENSE)
+
+
+Cumulus is a set of tools for writing Substrate-based Polkadot parachains.
+
+## Upstream Dependencies
+
+Below are the primary upstream dependencies utilized in this project:
+
+- [parity-scale-codec](https://crates.io/crates/parity-scale-codec)
+- [parity-db](https://crates.io/crates/parity-db)
+- [parity-common](https://github.com/paritytech/parity-common)
+- [trie](https://github.com/paritytech/trie)
+
```

Comment:
I think we should still have a Security section; it looks like the security file is not linked in the contributing guidelines.  
Maybe just copy the text from the substrate README?

Reply:
done